### ISSUE TITLE
Silo Trigger Nil value mode 1

### ISSUE THREAD

**Issue by pops64** [Courseplay/courseplay#1714]
v.58 Log spit out
```
C:/Users/Ryan/Documents/My Games/FarmingSimulator2017/mods/ZZZ_Courseplay5/toolManager.lua:700: attempt to index field 'currentSiloTrigger' (a nil value)
Error: Running LUA method 'update'.
``` 
Had 2 trailers. The first one was farmtek tdk 1600 the second one was the agroliner hkd 302. When it finished filling the first one is when I believe the error occurred. Has the fill animation did not stop between trailers. I had stop at next trigger active.(Though it did stop for the second trailer. And when it finished filling it did not update waypoint to the second one so it tried looping around back to the first waypoint)

Edited: Happened again not sure how to replicate it. This time the 2nd trailer was 75% full. If I can get you a save game I will if you need it 

Edited v2: Can replicate everytime. It seems on first load of game this error occurs in this configuration has seen in this [video](https://www.dropbox.com/s/i33yvyxjsjz340l/Farming%20Simulator%2017%202_6_2017%201_54_20%20PM.mp4?dl=0) Seems to be any 2+ trailer configuration has I tried flipping the trailers and got the same bug. And it is now line 709 instead of 700

**Comment by Satissis** [comment]
Should be fixed in v01.0071.
Try it out and report back.

**Comment by pops64** [comment]
All Good. Thanks
